test/TestCaseReader.cpp

```diff
@@ -37,6 +37,13 @@ TestCaseReader::TestCaseReader(string const& _filename):
 	m_unreadSettings = m_settings;
 }
 
+TestCaseReader::TestCaseReader(istringstream const& _str)
+{
+	tie(m_sources, m_lineNumber) = parseSourcesAndSettingsWithLineNumber(
+		static_cast<istream&>(const_cast<istringstream&>(_str))
```

Comment:
Why is it passed by `const&`?

Reply:
Because the test case reader should only read from the input stream and not modify it. This is similar to the other test case reader constructor that accepts a const string (that stores test file name)

Reply:
I think this should be fixed at some point...